flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/PartitionTestUtils.java

```diff
@@ -45,4 +52,14 @@ public static ResultPartition createPartition(
 			.setNumberOfSubpartitions(numChannels)
 			.build();
 	}
+
+	static void verifyCreateSubpartitionView(ResultPartitionManager partitionManager, ResultPartition partition) throws IOException {
```

Comment:
rename to `verifyCreateSubpartitionViewThrowsException`; consider passing in the result of `partition#getPartitionId()` instead.
The partition itself is not used, and as it stands the method is a a bit ambiguous since it could just as well call `ResultPartition#createSubpartitionView`